### ISSUE TITLE
website_sale_search_clear not working in 10c

### ISSUE THREAD

**Issue by lisandi** [it-projects-llc/website-addons#190]
As soon as we install website_sale_search_clear we receive a 500 error

**Comment by TimLai125** [comment]
@lisandi Hi, I have the same issue as well and I believe the issue is related to the inherited `shop` method
https://github.com/it-projects-llc/website-addons/blob/f3d2ffee6fc612c7bd7398d763fd1c3aac5b1295/website_sale_search_clear/controllers.py#L8-L12

The method should have `ppg` as the parameter
i.e.
```
@http.route()
def shop(self, page=0, category=None, search='', ppg=False, **post):
	if category and search:
		category = None
	return super(WebsiteSale, self).shop(page, category, search, ppg, **post)
```

**Comment by yelizariev** [comment]
Fixed in #300 for version 10.0
